sys-apps/attr/attr-2.5.1.recipe

```diff
@@ -0,0 +1,86 @@
+SUMMARY="Extended attributes tools"
+DESCRIPTION="Tools and libraries for working with filesystem extended attributes(xattr)."
+HOMEPAGE="https://savannah.nongnu.org/projects/attr"
+COPYRIGHT="2000-2005 Silicon Graphics, Inc
+	2001-2022 Andreas Gruenbacher"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://download.savannah.nongnu.org/releases/attr/attr-$portVersion.tar.gz"
+CHECKSUM_SHA256="bae1c6949b258a0d68001367ce0c741cebdacdd3b62965d17e5eb23cd78adaf8"
+PATCHES="attr-$portVersion.patchset"
+
+ARCHITECTURES="all"
+
+GLOBAL_WRITABLE_FILES="
+	settings/xattr.conf keep-old
+	"
+
+PROVIDES="
+	attr = $portVersion
+	cmd:attr
+	cmd:getfattr
+	cmd:setfattr
+	lib:libattr
```

Comment:
version for the lib is missing

Reply:
I've added the versions and redone the patch to be a bit more proper.